lua/lazyvim/plugins/extras/lang/erlang.lua

```diff
@@ -0,0 +1,20 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "erlang" },
+      root = "rebar.config",
```

Comment:
shouldn't root include ` 'erlang.mk'`?

Reply:
you're right. almost all the repos I've dealing with uses rebar3 now, but we should add erlang.mk. Already updated this branch